README.md

```diff
@@ -44,6 +67,9 @@ Meet with your corresponding group again. And, again, select which set will be D
 
 ***Q1.4 Designers: Briefly summarize their feedback for each solution:***
 
+Feedback for solution 1: too hard to teleport a player (technically killing them) and don't want to hurt morale
+Feedback for solution 2: cameras are necessary on players' heads to transmit VR, could inhibit game play
```

Comment:
If smaller, more compact VR cameras could somehow be installed as a patch on a jersey or headband, then something like this could definitely be more feasible and not harmful to gameplay.